plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2186,6 +2185,14 @@ So if getMinSpeed() returns null we fall back to getSpeed().
 
         vm.addComp(devices);
 
+        // Add extra configuration to User VM Domain XML before starting
+        if (vmTO.getType() == VirtualMachine.Type.User) {
+            String extraConfig = vmTO.getExtraConfig();
+            if (!Strings.isNullOrEmpty(extraConfig)){
```

Comment:
Can you replace this for `StringUtils.isNotBlank()' method on Apache Commons package?

Reply:
Done

Reply:
Done